app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
I'm not sure about this one so you double check it and decide.
from the conditions for the task:
![NVIDIA_Share_bxUttUgNHJ](https://github.com/mate-academy/py-elves-and-dwarves/assets/13889031/d1805e21-f13c-4620-974b-45c9a2bcd24c)

what chat GPT says: ```In Python, a class becomes abstract if it directly inherits from an ABC (Abstract Base Class) and implements at least one abstract method defined in that ABC```

So I belive that this class doesn't count as the abstract one because we didn't use any abstract methods from the parent ```Player``` class inside it, and you need to change it ```class Dwarf(Player, ABC):```
Again I'm not pro so double check it.